Cargo.toml

```diff
@@ -21,7 +21,7 @@ bellpepper-core = { workspace = true }
 bellperson = { workspace = true }
```

Comment:
Are the formatting changes included here produced by a tool, and if so, can we put it in CI? If not, can we revert them? I'd like to avoid hundred of lines of diff per month on whitespace adjustments.

Reply:
See #767